Build fails on latest CMake version

### Original bug

Yes

### Operating system

Linux

### Bug description

When trying to build using the `relive-git` AUR PKGBUILD on the latest version of CMake, i.e. `4.0.1` at this time, CMake throws an incompatibility error:
```
CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 has been removed from CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.

  Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.
```

One potential fix would be to raise the 3.2 minimum CMake requirement to 3.5, although I am not well-versed enough in the build process to know if that could break stuff.

### Repro steps

1. Launch build script
2. Get error

### Save file

_No response_

### Log

```
==> Making package: relive-git r5939.d797b6180-1 ([redacted for privacy])
==> Retrieving sources...
==> Validating source files with md5sums...
    relive ... Skipped
==> Making package: relive-git r5939.d797b6180-1 ([redacted for privacy])
==> Checking runtime dependencies...
==> Checking buildtime dependencies...
==> WARNING: Using existing $srcdir/ tree
==> Starting pkgver()...
==> Starting build()...
Submodule '3rdParty/Detours' (https://github.com/paulsapps/Detours.git) registered for path '3rdParty/Detours'
Submodule '3rdParty/googletest' (https://github.com/google/googletest.git) registered for path '3rdParty/googletest'
Submodule '3rdParty/json' (https://github.com/nlohmann/json.git) registered for path '3rdParty/json'
Submodule '3rdParty/jsonxx' (https://github.com/hjiang/jsonxx) registered for path '3rdParty/jsonxx'
Submodule '3rdParty/lodepng' (https://github.com/lvandeve/lodepng.git) registered for path '3rdParty/lodepng'
Submodule '3rdParty/magic_enum' (https://github.com/Neargye/magic_enum.git) registered for path '3rdParty/magic_enum'
Submodule '3rdParty/stb' (https://github.com/nothings/stb.git) registered for path '3rdParty/stb'
Cloning into '/build/relive-git/src/relive/3rdParty/Detours'...
Cloning into '/build/relive-git/src/relive/3rdParty/googletest'...
Cloning into '/build/relive-git/src/relive/3rdParty/json'...
Cloning into '/build/relive-git/src/relive/3rdParty/jsonxx'...
Cloning into '/build/relive-git/src/relive/3rdParty/lodepng'...
Cloning into '/build/relive-git/src/relive/3rdParty/magic_enum'...
Cloning into '/build/relive-git/src/relive/3rdParty/stb'...
Submodule path '3rdParty/Detours': checked out '5e081ec7a295b8b090392f579e4dd75e997b88bb'
Submodule path '3rdParty/googletest': checked out '98a0d007d7092b72eea0e501bb9ad17908a1a036'
Submodule path '3rdParty/json': checked out '293f67f9ff1a3c7d16cf0959c95761e89b9b64e9'
Submodule path '3rdParty/jsonxx': checked out '0b8e472a10b10af14b0a214c6377f9142f4f70a5'
Submodule path '3rdParty/lodepng': checked out '7fdcc96a5e5864eee72911c3ca79b1d9f0d12292'
Submodule path '3rdParty/magic_enum': checked out '58833fa067361a769892149b27b06cf60d89bf46'
Submodule path '3rdParty/stb': checked out 'c9064e317699d2e495f36ba4f9ac037e88ee371a'
CMake Error at CMakeLists.txt:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 has been removed from CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.

  Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.


-- Configuring incomplete, errors occurred!
==> ERROR: A failure occurred in build().
    Aborting...
==> ERROR: Build failed, check /home/[redacted for privacy]/chroot/[redacted for privacy]/build
error: failed to build 'relive-git-r3641.fcce1105-3': failed to run: makechrootpkg -r /home/[redacted for privacy]/chroot -d /var/cache/pacman/pkg/ -- -feA --noconfirm --noprepare --holdver: 
error: packages failed to build: relive-git-r3641.fcce1105-3
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.2 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.10 FATAL_ERROR)
 project(relive VERSION 0.1 LANGUAGES C CXX)
 
 set(CMAKE_CXX_STANDARD 17)

--- a/options.cmake
+++ b/options.cmake
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.2 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.10 FATAL_ERROR)
 
 option(RENDER_TEST "Create test object that renders all prim types on boot" OFF)
 option(DEVELOPER_MODE "Boot direct to main selection screen, enable debug.sav loading" OFF)

